resources/cdk/aurora_serverless_app/README.md

```diff
@@ -0,0 +1,137 @@
+# Aurora serverless sample application resources
+
+## Overview
+
+Creates AWS resources for Amazon Aurora sample applications. The scripts in this
+example create the following resources:
+ 
+* An AWS Secrets Manager secret that contains administrator credentials in a format 
+that can be used by an Aurora MySQL database. 
+* An Aurora MySQL database configured to use the credentials from the secret.
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Deploying resources
+
+You can use the AWS Cloud Development Kit (AWS CDK) or the AWS Command Line Interface
+(AWS CLI) to deploy and destroy the resources for this example.
+
+### Deploying with the AWS CDK
+
+To deploy with the AWS CDK, you must install [Node.js](https://nodejs.org) and the 
+[AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html).
+
+This example was built and tested with AWS CDK 2.25.0.
+
+Deploy AWS resources by running the following at a command prompt in the
+`resources/cdk/aurora_serverless_app` folder:
+
+```
+npm install
+cdk deploy
+```
+
+The stack takes a few minutes to deploy. When it completes, it prints output like 
+the following:
+
+```
+Outputs:
+doc-example-aurora-app.ClusterArn = arn:aws:rds:us-west-2:0123456789012:cluster:doc-example-aurora-app-docexampleauroraappcluster-1bqmf5EXAMPLE
+doc-example-aurora-app.DbName = auroraappdb
+doc-example-aurora-app.SecretArn = arn:aws:secretsmanager:us-west-2:0123456789012:secret:docexampleauroraappsecret8B-rEHdtEXAMPLE-111222
+```
+
+You can destroy the resources by running the following:
+
+```
+cdk destroy
+```
+
+### Deploying with the AWS CLI 
+
+To deploy with the AWS CLI you must first install the 
+[AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+Deploy AWS resources by running the following at a command prompt in the 
+`resources/cdk/aurora_serverless_app` folder:
+
+```
+aws cloudformation create-stack --template-body file://setup.yaml --stack-name YOUR_STACK_NAME
+```
+
```

Comment:
Suggest adding this note:
Important: The stack name must be unique within an AWS Region and AWS account. You can specify up to 128 characters, and numbers and hyphens are allowed.

Reply:
fixed